### ISSUE TITLE
ModuleNotFoundError: No module named 'dataset_pair'

### ISSUE THREAD

**Issue by qingchenkanlu** [NOrangeeroli/SecondPose#7]
No module named 'dataset_pair'. The dataset_pair.py has not been uploaded?

**Comment by hiyyg** [comment]
@qingchenkanlu Have you solved this problem?

**Comment by qingchenkanlu** [comment]
I replace dataset_pair to dataset, the train code can run. However, I cannot fully reproduce the results presented in the paper.
2024-07-08 15:07:37,689 - ************************ Start Logging ************************
2024-07-08 15:13:15,914 - 3D IoU at 25: 84.2
2024-07-08 15:13:15,915 - 3D IoU at 50: 83.9
2024-07-08 15:13:15,915 - 3D IoU at 75: 75.6
2024-07-08 15:13:15,915 - 5 degree, 2cm: 52.6
2024-07-08 15:13:15,915 - 5 degree, 5cm: 63.5
2024-07-08 15:13:15,915 - 10 degree, 2cm: 68.5
2024-07-08 15:13:15,915 - 10 degree, 5cm: 86.3
2024-07-08 15:13:53,300 - 3D IoU at 25: 84.0
2024-07-08 15:13:53,300 - 3D IoU at 50: 80.8
2024-07-08 15:13:53,301 - 3D IoU at 75: 40.5

If someone manages to reproduce the results of the paper in the future, they can provide the pre-trained weights.

**Comment by hiyyg** [comment]
> : 52.6
> 2024-07-08 15:13:15,915 - 5 degree, 5cm: 63.5
> 2024-07-08 15:13:15,915 - 10 degree, 2cm: 68.5
> 2024-07-08 15:13:15,915 - 10 degree, 5cm: 86.3

Thanks for your information! Did you run VI-Net before? If so, could you reproduce the training results of VI-Net? It seems the ts network in this work is the same as that in the VI-Net work.

**Comment by qingchenkanlu** [comment]
No, I have not. VI-Net provides complete experimental results and training weights. I consider the results to be reliable.
SecondPose is developed based on VI-Net, ts network is similar.

**Comment by hiyyg** [comment]
Well, I could not reproduce the training results of VI-Net, although their provided weights have close results to their paper.

**Comment by shuaiqiqi1227** [comment]
Hello, data_preprocessing.py requires the data set camera25 and real275, but the data set format in SelF-DPDN does not have this, and the data set also requires the detection folder. What should I do
